crates/re_types_builder/src/codegen/rust/api.rs

```diff
@@ -505,19 +502,39 @@ fn quote_doc_from_docs(reporter: &Reporter, docs: &Docs) -> TokenStream {
 
     let mut lines = crate::codegen::get_documentation(docs, &["rs", "rust"]);
 
-    let examples = collect_examples(docs)
+    let examples = collect_examples(docs, "rs", true)
         .map_err(|err| reporter.error(err))
         .unwrap_or_default();
     if !examples.is_empty() {
         lines.push(" ".into());
-        let section_title = if examples.count == 1 {
+        let section_title = if examples.len() == 1 {
             "Example"
         } else {
             "Examples"
         };
         lines.push(format!(" ## {section_title}"));
         lines.push(" ".into());
-        lines.extend(examples.lines.into_iter().map(|line| format!(" {line}")));
+        let mut examples = examples.into_iter().peekable();
+        while let Some(example) = examples.next() {
+            if let Some(title) = example.base.title {
+                lines.push(format!(" ### {title}"));
+            }
+            lines.push(" ```ignore".into());
```

Comment:
Why `ignore`? All the code examples should properly parse.

`no_run` should work just as well

```suggestion
            lines.push(" ```no_run".into());
```

Reply:
`ignore` sounds correct to me.

All of these are already compiled and ran as part of the `code_examples` crate; recompiling them here would only slow down the CI for very little benefit?

Reply:
fair; I guess we don't need the extra sanity check